exatomic/gaussian/output.py

```diff
@@ -608,6 +619,212 @@ def parse_momatrix(self):
         if bcoefs is not None:
             self.momatrix['coef1'] = bcoefs
 
+    def parse_frequency_ext(self):
+        '''
+        Parses extended frequency data present in FChk file
+
+        Note:
+            Requires Freq(SaveNormalModes) in input
+        '''
+        # Frequency regex
+        _renmode = 'Number of Normal Modes'
+        _redisp = 'Vib-Modes'
+        _refinfo = 'Vib-E2'
+        _remass = 'Vib-AtMass'
+        # TODO: find out what these two values are useful for
+        _rendim = 'Vib-NDim'
+        _rendim0 = 'Vib-NDim0'
+
+        found = self.find(_renmode)
+        if not found:
+            return
+        else:
+            found = self.find(_renmode, _redisp, _refinfo, _remass, keys_only=True)
+        # find number of vibrational modes
+        nmode = self._intme(found[_renmode])
+        # get extended data (given by mapper array)
+        all_info = self._dfme(found[_refinfo], nmode * 14)
+        all_info[abs(all_info) < self.tol] = 0
+        # mapper array to filter through all of the values printed on the fchk file
+        # TODO: have to find labels of unknown columns
+        mapper = ["freq", "r_mass", "f_const", "ir_int", "unk1", "unk2", "unk3",
+                  "dipole_s", "rot_s", "unk4", "unk5", "unk6", "unk7", "em_angle"]
+        # build extended frequency table
+        self.frequency_ext = pd.DataFrame.from_dict({mapper[int(i/nmode)]: all_info[i:i+nmode]
```

Comment:
Would integer division work here (`//`) instead of the cast to int? Arguably cleaner if it does.

Reply:
Would that cause any issues with the python 2.x release? I found something saying we may need to import from future?

Reply:
Yeah you're right I think. My mistake.